Changes.md

```diff
@@ -9,6 +9,7 @@ Improvements
   - Changed default value for `principled_bsdf.specular_ior_level` to `0.5`, matching Blender.
   - Added support for `uv.tangent` and `uv.tangent_sign` primitive variables to assist in rendering with normal maps (#5269).
 - AttributeQuery, PrimitiveVariableQuery, ContextQuery, OptionQuery, ShaderQuery : Added support for querying arrays of length 1 as their equivalent scalar types.
+- USD : Added automatic render-time translation of UsdPreviewSurface shaders to 3Delight.
```

Comment:
I'd be inclined to bill this as `3Delight : ` rather than `USD :`.

Reply:
Billed as 3Delight in f0b1b25.